src/assets/styles/30-components/__manifest.scss

```diff
@@ -7,6 +7,7 @@
 
 @import './src/components/Button/button';
 @import './src/components/Grid/grid';
+@import './src/components/Grid/Cell/grid-cell';
```

Comment:
With changes to https://github.com/wellcometrust/corporate-components/pull/125 should this now be `@import './src/components/Grid/grid-cell';` ?

Reply:
nope - GridCell is still in a sub-folder (of Grid), it just has an index file allowing it to be imported as `@import GridCell` (rather than `GridCell/GridCell`) 🤓